Add initial trpc interface package (#780)
webstudio-is/webstudio-designer: packages/trpc-interface/README.md

# trpc-interface

- `trpc-interface` can implement various TRPC server routers.

> - it can act as a proxy for TRPC services implemented at SAAS (that services must have the same API interface).

Reviewer: ```suggestion
- it can act as a proxy for TRPC services implemented in any other apps (those services must have the same API interface).
```
